### PR TITLE
CNFT2-1557 Revise Search and add condition title

### PR DESCRIPTION
**Pull request by jlee-enquizit**
## Description

Here we update some of the buttons and UX on Add new page.

## Tickets

[CNFT2-1557](https://cdc-nbs.atlassian.net/browse/CNFT2-1557)

## Steps to reproduce
1. On Pages Library, select `Add new page`
2. Select "Investigation" from the Event type menu
3. Buttons and layout should match designs. https://www.figma.com/file/Uz9YzHbHQAubHpQTcsibA2/NBS-PageBuilder-2.0?type=design&node-id=1-6365&mode=dev

## Checklist before requesting a review
- [ x] PR focuses on a single story
- [x ] Code has been fully tested to meet acceptance criteria
- [ x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ x] All new functions/classes/components reasonably small
- [ x] Functions/classes/components focused on one responsibility
- [ x] Code easy to understand and modify (clarity over concise/clever)
- [ x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ x] PR does not contain hardcoded values (Uses constants)
- [ x] All code is covered by unit or feature tests

[CNFT2-1557]: https://cdc-nbs.atlassian.net/browse/CNFT2-1557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by achapm — APPROVED** [review]
👍

File: apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPage.scss [hunk 1]
```diff
@@ -40,8 +44,9 @@
             p {
                 font-size: 0.875rem;
                 line-height: 1.4rem;
-                margin-top: 0.5rem;
-                color: colors.$base-light;
+                margin-top: 1rem;
+                margin-bottom: 0.5rem;
+                color: colors.$base-light;-size: 0.75rem;
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPage.scss:49]: Suggestion (non-blocking): was something left out? I see a dangling `-size: 0.75 rem;`

**Review by mpeels — APPROVED** [review]
Please see comment above, otherwise good to go.

File: apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPage.scss (hunk 1 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/apps/page-builder/pages/AddNewPage/AddNewPage.scss]: fixed it

**Review by mpeels — APPROVED** [review]

**Pull request merged**